src/Dimensions/merged.jl

```diff
@@ -1,3 +1,5 @@
+abstract type AbstractMergedLookup{T} <: Lookup{T,1} end
```

Comment:
I think this name can be better, maybe `MultiDimensionalLookup`

Reply:
`multidims` is used as the variable name above...

Reply:
Yeah multidimensional sounds good.

Reply:
should this even be 1D then?

Reply:
Yeah thats kind of confusting really, everything is still 1d in practice